src/sagemaker_containers/_env.py

```diff
@@ -522,6 +522,7 @@ def __init__(self, resource_config=None, input_data_config=None, hyperparameters
 
         self._hosts = hosts
         self._network_interface_name = network_interface_name
+        self._parameter_server_num = sagemaker_hyperparameters.get(_params.PARAMETER_SERVER_NUM_PARAM)
```

Comment:
It's ok to do that in SageMaker containers if parameter servers is a common concept across all containers. We might have to change PyTorch and Chainer to be able to do that. I guess, we will have to get feedback from the team first.